database/migrations/2021_02_11_082928_add_favorite_status_to_favorites_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddFavoriteStatusToFavoritesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('favorites', function (Blueprint $table) {
+            //
+            $table->unsignedBigInteger('posted_status');
+            $table->foreign('posted_status')->references('id')->on('favorite_status');
```

Comment:
change field name from "posted_status" to "favorite_status" and make necessary changes